kata-design-requirements.md

```diff
@@ -0,0 +1,82 @@
+## Mandatory requirements
+
+The Kata Containers runtime **MUST** fulfill all requirements below:
+
+### OCI compatibility
+The Kata Containers runtime **MUST** implement the OCI runtime specification and support all the OCI runtime operations.
+
+### `runc` CLI compatibility
+In theory, being OCI compatible should be enough. In practice the a Kata Containers runtime should comply the latest *stable* `runc` CLI. In particular, it **MUST** implement with the following `runc` commands:
```

Comment:
This document appears to be partially written to conform to [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt). However, it needs certain words to be emphasised such as "should" (should be **SHOULD**, etc).